mirdata/beatles.py

```diff
@@ -16,6 +32,22 @@
 
 
 class Track(object):
+    """Beatles track class
+
+    Args:
+        track_id (str): track id of the track
+        data_home (str): data home folder path
```

Comment:
four juxtaposed nouns? what do they mean?

Reply:
i was being german but @rabitt reviewed it above, i'm changing it now.